webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBarReducer.test.js

```diff
@@ -47,8 +48,5 @@ const fixtures = {
 };
 
 describe('BreadcrumbBar reducer', () => {
-  const reduce = ({ state, action = {} } = {}) => reducer(state, action);
-
-  Object.entries(fixtures).forEach(([description, action]) =>
-    it(description, () => expect(reduce(action)).toMatchSnapshot()));
+  testReducerSnapshotWithFixtures(reducer, fixtures);
```

Comment:
Not that important but it can be a one-line
```js
describe('BreadcrumbBar reducer', () => testReducerSnapshotWithFixtures(reducer, fixtures));
// or
describe('BreadcrumbBar reducer', () =>
  testReducerSnapshotWithFixtures(reducer, fixtures));
```